llmware/library.py

```diff
@@ -702,7 +1093,18 @@ def add_dialogs(self, input_folder=None):
 
     def add_image(self, input_folder=None):
 
-        """ Main method to add image and scanned OCR content into a library """
+        """ Main method to add image and scanned OCR content into a library 
```

Comment:
Delete space before you start the doc and add full stop.
```suggestion
        """Main method to add image and scanned OCR content into a library.
```